### PR TITLE
improved, centralized and darkened the search bar (issue-177)

### PR DESCRIPTION
**Pull request by iamatifmoin**
## Summary

I have darkened the colour of the search bar and centralized it. 
I have also added a hover effect, which darkens the colour when the user hovers over it.
Helps in #177

## Changes

- Centralized the search bar
- Darkened the colour of the search bar
- Added hover effect to the search bar

## Screenshots

darkened search bar
Zubhub

hover effect
Zubhub (1)

### CODE REVIEW

File: zubhub_frontend/zubhub/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```
Comment (tuxology) [inline zubhub_frontend/zubhub/package.json]: Why do we need this?

File: zubhub_frontend/zubhub/src/views/PageWrapper.jsx [hunk 2]
```diff
@@ -752,10 +756,7 @@ function PageWrapper(props) {
                   </InputLabel>
                   <Autocomplete
                     options={options}
-                    defaultValue={
-                      props.location.search &&
-                      getQueryParams(window.location.href).get('q')
-                    }
+                    defaultValue={getQueryParams(window.location.href).get('q')}
```
Comment (tuxology) [inline zubhub_frontend/zubhub/src/views/PageWrapper.jsx]: Why remove the props here?

**Review by tuxology — COMMENTED** [review]
Left a few comments, we can approve one we are comfortable with reasoning

File: zubhub_frontend/zubhub/package.json (hunk 1 above)
Comment (iamatifmoin) [reply zubhub_frontend/zubhub/package.json]: using `"start": "react-scripts start"` generated the following error for me:
> Error: error:0308010C:digital envelope routines::unsupported

I looked for solutions [here](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported) and found that the error can be resolved by using `"start": "react-scripts --openssl-legacy-provider start"`

This change, however, should have been made locally only and should not have been committed.
I will reset the change by making another commit.

File: zubhub_frontend/zubhub/src/views/PageWrapper.jsx (hunk 2 above)
Comment (iamatifmoin) [reply zubhub_frontend/zubhub/src/views/PageWrapper.jsx]: I think I made this change unintentionally while trying to figure out the working of the code. 

I will reset the change in a new commit and make sure to go through the changes I make next time.

**Review by tuxology — APPROVED** [review]

**Comment by iamatifmoin** [comment]
@tuxology 
When will the PR be merged? Or are some changes required?

**Pull request merged**